internal/pkg/cli/app_init.go

```diff
@@ -256,6 +273,31 @@ func (o *initAppOpts) askDockerfile() error {
 	return nil
 }
 
+func (o *initAppOpts) askAppPort() error {
+	if o.AppPort != 0 {
+		return nil
+	}
+
+	port, err := o.prompt.Get(
+		fmt.Sprintf(fmtAppInitAppPortPrompt),
+		fmt.Sprintf(fmtAppInitAppPortHelpPrompt),
+		validateApplicationPort,
+		prompt.WithDefaultInput("80"),
```

Comment:
Just a nit: should we make this "80" a constant so that probably in the future when we want to change the default port it would be easier and also we might have different default port for the other kinds of app.